tests/Unit/Domain/Test_Orientations.cpp

```diff
@@ -0,0 +1,181 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
```

Comment:
Except factory generated things where the idea doesn't apply. However, if the derived classes have copy and move semantics they should be comparable.